consensus/vbft/block_pool.go

```diff
@@ -322,14 +322,14 @@ func (pool *BlockPool) endorseDone(blkNum uint32, C uint32) (uint32, bool, bool)
 		for _, esig := range eSigs {
 			if esig.ForEmpty {
 				emptyEndorseCount++
-				if emptyEndorseCount > int(C) {
+				if emptyEndorseCount >= int(C) {
```

Comment:
in your last change, there'll C+1 proposers, the last proposer can also be endorser.
So, there should always be at least C+1 honest endorsers.
No change need here.

Reply:
already fixed